Document $suffix argument to tempnam()

jdufresne opened the pull request.

jdufresne committed:
Document $suffix argument to tempnam()

OskarStark reviewed (approved):
I guess it must be merged in 3.4, but we take care of it 👍🏻👌🏻

pierredup commented on components/filesystem.rst:
```diff
@@ -284,6 +284,8 @@ tempnam
     // returns a path like : /tmp/prefix_wyjgtF
     $filesystem->tempnam('/tmp', 'prefix_');
+    // returns a path like : /tmp/prefix_wyjgtF.png
+    $filesystem->tempnam('/tmp', 'prefix_', '.png');
```
I think you can add a block below specifying the version this was introduced in

```
.. versionadded:: 4.4

    The option to set a suffix in  ``tempnam()`` was introduced in Symfony 4.4.
```

OskarStark replied:
Yes please 👍🏻

jdufresne commented:
Thanks. I added the `versionadded`.

javiereguiluz committed:
[Form] Documented the choice_self_translation option of LanguageType

javiereguiluz committed:
Merge branch '4.4' into 5.0

* 4.4:
  fix a typo in mailer docs (Mutliple -> Multiple)

javiereguiluz committed:
minor #12750 [Form] Documented the choice_self_translation option of LanguageType (javiereguiluz)

This PR was squashed before being merged into the master branch (closes #12750).

Discussion
----------

[Form] Documented the choice_self_translation option of LanguageType

Fixes #12745.

Commits
-------

7361fdbf9 [Form] Documented the choice_self_translation option of LanguageType

javiereguiluz committed:
Merge branch '4.4' into 5.0

* 4.4:
  [Cache] Fixed a minor RST syntax issue
  [Cache] Expand the docs about the cache chain
  Explain the prefix used for expressions in YAML files
  Minor tweak in a security expression explanation

javiereguiluz committed:
Merge branch '5.0'

* 5.0:
  [Cache] Fixed a minor RST syntax issue
  [Cache] Expand the docs about the cache chain
  Explain the prefix used for expressions in YAML files
  Minor tweak in a security expression explanation

wouterj committed:
Merge branch '4.4' into 5.0

* 4.4:
  Revert "Removed 4.4 versionadded from the 4.3 docs"
  Removed 4.4 versionadded from the 4.3 docs
  Update setup.rst
  Update messenger.rst

wouterj committed:
Merge branch '5.0'

* 5.0:
  Revert "Removed 4.4 versionadded from the 4.3 docs"
  Removed 4.4 versionadded from the 4.3 docs
  Update setup.rst
  Update messenger.rst

javiereguiluz committed:
Merge branch '4.4' into 5.0

* 4.4:
  Updated a link to Debug component
  Remove ErrorRenderer reference everywhere

javiereguiluz committed:
Merge branch '5.0'

* 5.0:
  Updated a link to Debug component
  Remove ErrorRenderer reference everywhere

javiereguiluz committed:
Merge branch '4.4' into 5.0

* 4.4:
  Updated the location of the errors.xml file

javiereguiluz committed:
Merge branch '5.0'

* 5.0:
  Updated the location of the errors.xml file

javiereguiluz committed:
Merge branch '4.4' into 5.0

* 4.4:
  Minor tweaks
  Fix 12273 Added new ErrorController

javiereguiluz committed:
Merge branch '5.0'

* 5.0:
  Minor tweaks
  Fix 12273 Added new ErrorController

javiereguiluz committed:
Removed 4.4 versionadded and deprecations

javiereguiluz committed:
Merge branch '5.0'

* 5.0:
  Removed 4.4 versionadded and deprecations

javiereguiluz committed:
Merge branch '4.4' into 5.0

* 4.4:
  Mentioned the PHP 7.4 class preloading
  Scalingo is another PaaS which supports Symfony

javiereguiluz committed:
Removed the versionadded directive

javiereguiluz committed:
Merge branch '5.0'

* 5.0:
  Removed the versionadded directive
  Mentioned the PHP 7.4 class preloading
  Scalingo is another PaaS which supports Symfony

javiereguiluz committed:
Merge branch '4.4' into 5.0

* 4.4:
  ListenerInterface deprecated since Symfony 4.3

javiereguiluz committed:
Merge branch '5.0'

* 5.0:
  ListenerInterface deprecated since Symfony 4.3

javiereguiluz committed:
Merge branch '4.4' into 5.0

* 4.4:
  Fix highlighting. Refs #12781

javiereguiluz committed:
Merge branch '5.0'

* 5.0:
  Fix highlighting. Refs #12781

javiereguiluz committed:
Merge branch '4.4' into 5.0

* 4.4:
  Update setup.rst

javiereguiluz committed:
Merge branch '5.0'

* 5.0:
  Update setup.rst

javiereguiluz committed:
Merge branch '4.4' into 5.0

* 4.4:
  added minor review of example code

javiereguiluz committed:
Merge branch '5.0'

* 5.0:
  added minor review of example code

javiereguiluz committed:
Merge branch '4.4' into 5.0

* 4.4:
  Add mandatory exit status
  Update console.rst

javiereguiluz committed:
Removed the deprecated note

javiereguiluz committed:
Merge branch '5.0'

* 5.0:
  Add mandatory exit status
  Update console.rst

javiereguiluz committed:
Merge branch '4.4' into 5.0

* 4.4:
  Mention twig/extensions is not for Twig 3

javiereguiluz committed:
Merge branch '5.0'

* 5.0:
  Mention twig/extensions is not for Twig 3

javiereguiluz committed:
Merge branch '4.4' into 5.0

* 4.4:
  Add note about .env.local being ignored in test environment

javiereguiluz committed:
Merge branch '5.0'

* 5.0:
  Add note about .env.local being ignored in test environment

javiereguiluz committed:
Merge branch '4.4' into 5.0

* 4.4:
  Update readme
  Fixed the minimum Twig version needed for lazy extensions

javiereguiluz committed:
Merge branch '5.0'

* 5.0:
  Tweaks
  Fixed the minimum Twig version needed for lazy extensions

javiereguiluz committed:
Merge branch '4.4' into 5.0

* 4.4:
  Fixed code indentation

javiereguiluz committed:
Merge branch '5.0'

* 5.0:
  Fixed code indentation

jdreesen committed:
Fix output of before() example

jdreesen committed:
Fix UnicodeString::fromCodePoints example

jdreesen committed:
Remove superfluous space

OskarStark committed:
minor #12792 Remove superfluous space (jdreesen)

This PR was merged into the 5.0 branch.

Discussion
----------

Remove superfluous space

<!--

If your pull request fixes a BUG, use the oldest maintained branch that contains
the bug (see https://symfony.com/roadmap for the list of maintained branches).

If your pull request documents a NEW FEATURE, use the same Symfony branch where
the feature was introduced (and `master` for features of unreleased versions).

-->

Commits
-------

ff0a9230c Remove superfluous space

OskarStark committed:
Merge branch '4.4' into 5.0

* 4.4:
  Typo in command name. refs #12785
  Fix build error
  [Mailer] Mentioned the getMessageId() method
  [DI] Documented the lint:container command

OskarStark committed:
Removed old versionadded directives

OskarStark committed:
Merge branch '5.0'

* 5.0:
  Removed old versionadded directives
  Typo in command name. refs #12785
  Fix build error
  Remove superfluous space
  [Mailer] Mentioned the getMessageId() method
  [DI] Documented the lint:container command

javiereguiluz committed:
minor #12790 Fix `UnicodeString::before()` example (jdreesen)

This PR was merged into the 5.0 branch.

Discussion
----------

Fix `UnicodeString::before()` example

If I'm not mistaken, the result of `u('hello world')->before('o');` would be `'hell'` instead of `'hell '`.

Commits
-------

9322a00f6 Fix output of before() example

javiereguiluz committed:
minor #12791 Fix `UnicodeString::fromCodePoints()` example (jdreesen)

This PR was merged into the 5.0 branch.

Discussion
----------

Fix `UnicodeString::fromCodePoints()` example

`UnicodeString::fromCodePoints()` returns a `UnicodeString` so the next line `// $foo = 'नमस्ते'` is not correct as it implies the result to be a `string`.

Thus, the result should be casted to a `string` (or `->toString()` should be called) to match the comment.

Commits
-------

6793fbecc Fix UnicodeString::fromCodePoints example

javiereguiluz committed:
Minor tweak

javiereguiluz committed:
Merge branch '5.0'

* 5.0:
  Fix UnicodeString::fromCodePoints example
  Fix output of before() example

javiereguiluz committed:
Merge branch '4.4' into 5.0

* 4.4:
  Remove unused use statement

javiereguiluz committed:
Merge branch '5.0'

* 5.0:
  Remove unused use statement

javiereguiluz committed:
minor #12718 fix TraceableEventDispatcherInterface removed in 5.0 (jzawadzki)

This PR was merged into the 5.0 branch.

Discussion
----------

fix TraceableEventDispatcherInterface removed in 5.0

As per https://github.com/symfony/symfony/pull/24392/files in 5.0 `TraceableEventDispatcherInterface` is no more.

Commits
-------

771924399 fix TraceableEventDispatcherInterface removed in 5.0

javiereguiluz committed:
Merge branch '5.0'

* 5.0:
  fix TraceableEventDispatcherInterface removed in 5.0

javiereguiluz committed:
Merge branch '4.4' into 5.0

* 4.4:
  [Console] Mention that CommandTester doesn't dispatch console events

javiereguiluz committed:
Merge branch '5.0'

* 5.0:
  [Console] Mention that CommandTester doesn't dispatch console events

javiereguiluz committed:
Merge branch '4.4' into 5.0

* 4.4:
  Use a instead of an before consonant sounds

javiereguiluz committed:
Merge branch '5.0'

* 5.0:
  Use a instead of an before consonant sounds

javiereguiluz committed:
Merge branch '4.4' into 5.0

* 4.4:
  Tweak the explanation about event priorities

javiereguiluz committed:
Merge branch '5.0'

* 5.0:
  Tweak the explanation about event priorities

javiereguiluz committed:
Merge branch '4.4' into 5.0

* 4.4:
  Mention the issues of ETag-based validation and Apache compression

javiereguiluz committed:
Merge branch '5.0'

* 5.0:
  Mention the issues of ETag-based validation and Apache compression

javiereguiluz committed:
Merge branch '4.4' into 5.0

* 4.4:
  Middleware clarification about being called on \"receive\"

javiereguiluz committed:
Merge branch '5.0'

* 5.0:
  Middleware clarification about being called on \"receive\"

OskarStark committed:
Merge branch '4.4' into 5.0

* 4.4:
  [PropertyAccess] Update the method directives
  Tweaked the PropertyAccess doc
  Fixed method names when using the method directive

OskarStark committed:
Merge branch '5.0'

* 5.0:
  [PropertyAccess] Update the method directives
  Tweaked the PropertyAccess doc
  Fixed method names when using the method directive

OskarStark committed:
Merge branch '4.4' into 5.0

* 4.4:
  [Mailer] Minor tweaks in the mailer docs
  [Messenger] Minor tweaks for the main Messenger article
  [Messenger] Added a missing class import

OskarStark committed:
Merge branch '5.0'

* 5.0:
  [Mailer] Minor tweaks in the mailer docs
  [Messenger] Minor tweaks for the main Messenger article
  [Messenger] Added a missing class import

abdounikarim committed:
Update database.rst

Hello, 

I just add return type hint like the parent class : https://github.com/sebastianbergmann/phpunit/blob/master/src/Framework/TestCase.php

Cheers 😉

OskarStark committed:
Merge branch '4.4' into 5.0

* 4.4:
  Added the docs for the UUID polyfill
  [Validation] Mention the option to pass stringable objects
  [Routing] Documented the option to exclude patterns
  [Monolog] Add a caution note about Symfony Mailer and Monolog
  [ErrorHandler] Simplified docs

OskarStark committed:
[Routing] Removed old versionadded directive

OskarStark committed:
[Validator] Removed old versionadded directive

OskarStark committed:
Merge branch '5.0'

* 5.0:
  [Validator] Removed old versionadded directive
  [Routing] Removed old versionadded directive
  Added the docs for the UUID polyfill
  [Validation] Mention the option to pass stringable objects
  [Routing] Documented the option to exclude patterns
  [Monolog] Add a caution note about Symfony Mailer and Monolog
  [ErrorHandler] Simplified docs

javiereguiluz committed:
Merge branch '4.4' into 5.0

* 4.4:
  Update configuration.rst

javiereguiluz committed:
Merge branch '5.0'

* 5.0:
  Update configuration.rst

javiereguiluz committed:
Merge branch '4.4' into 5.0

* 4.4:
  Update references to 4.4 version

javiereguiluz committed:
Merge branch '5.0'

* 5.0:
  Update references to 4.4 version

javiereguiluz committed:
Merge branch '4.4' into 5.0

* 4.4:
  [Performance] Document the container.dumper.inline_factories parameter

javiereguiluz committed:
Merge branch '5.0'

* 5.0:
  [Performance] Document the container.dumper.inline_factories parameter

javiereguiluz committed:
minor #12824 Update database.rst (abdounikarim)

This PR was merged into the 5.0 branch.

Discussion
----------

Update database.rst

Hello,

I just add return type hint like the parent class : https://github.com/sebastianbergmann/phpunit/blob/master/src/Framework/TestCase.php

Cheers 😉

Commits
-------

9d1e2e038 Update database.rst

javiereguiluz committed:
Merge branch '5.0'

* 5.0:
  Update database.rst

javiereguiluz committed:
Merge branch '4.4' into 5.0

* 4.4:
  Update paragraph about portable command lines

javiereguiluz committed:
Merge branch '5.0'

* 5.0:
  Update paragraph about portable command lines

atailouloute committed:
Make custom routers compatible with the LoaderInterface

atailouloute committed:
Update guard_authentication.rst

xabbuh committed:
remove old versionadded directive

xabbuh committed:
minor #12859 remove old versionadded directive (xabbuh)

This PR was merged into the 5.0 branch.

Discussion
----------

remove old versionadded directive

Commits
-------

5ed10787c remove old versionadded directive

MrYamous committed:
Add documentation for new reverse() method

xabbuh committed:
minor #12856 Update guard_authentication.rst (atailouloute)

This PR was merged into the 5.0 branch.

Discussion
----------

Update guard_authentication.rst

Minor improvements

Commits
-------

7771a2bfa Update guard_authentication.rst

xabbuh committed:
Merge branch '4.4' into 5.0

* 4.4:
  [#12835] add use statement
  Update custom-transport.rst
  Update injection_types.rst
  Document PasswordAuthenticatedInterface
  Update Valid.rst
  Update password_migration.rst

xabbuh committed:
Merge branch '5.0'

* 5.0:
  [#12835] add use statement
  Update custom-transport.rst
  Update injection_types.rst
  Document PasswordAuthenticatedInterface
  Update Valid.rst
  remove old versionadded directive
  Update guard_authentication.rst
  Update password_migration.rst

fancyweb committed:
[String] Replace codePoint by codePointsAt

javiereguiluz committed:
Merge branch '4.4' into 5.0

* 4.4:
  Update security.rst

javiereguiluz committed:
Merge branch '5.0'

* 5.0:
  Update security.rst

javiereguiluz committed:
Merge branch '4.4' into 5.0

* 4.4:
  backport some tweaks from 5.0 docs

javiereguiluz committed:
Merge branch '5.0'

* 5.0:
  backport some tweaks from 5.0 docs

javiereguiluz committed:
Merge branch '4.4' into 5.0

* 4.4:
  Minor tweak

javiereguiluz committed:
Merge branch '5.0'

* 5.0:
  Minor tweak

javiereguiluz committed:
minor #12876 [String] Replace codePoint by codePointsAt (fancyweb)

This PR was merged into the 5.0 branch.

Discussion
----------

[String] Replace codePoint by codePointsAt

https://github.com/symfony/symfony/pull/33940 renamed the `codePoint` method to `codePointsAt`.

Commits
-------

d50779398 [String] Replace codePoint by codePointsAt

javiereguiluz committed:
Minor tweaks

javiereguiluz committed:
Merge branch '5.0'

* 5.0:
  [String] Replace codePoint by codePointsAt

javiereguiluz committed:
Merge branch '4.4' into 5.0

* 4.4:
  Replace references to Goutte with HttpBrowser

javiereguiluz committed:
Merge branch '5.0'

* 5.0:
  Replace references to Goutte with HttpBrowser

javiereguiluz committed:
Merge branch '4.4' into 5.0

* 4.4:
  SMimeEncrypter add Tip

javiereguiluz committed:
Merge branch '5.0'

* 5.0:
  SMimeEncrypter add Tip

javiereguiluz committed:
Merge branch '4.4' into 5.0

* 4.4:
  add url to bootstrap.php for older versions

javiereguiluz committed:
Merge branch '5.0'

* 5.0:
  add url to bootstrap.php for older versions

javiereguiluz committed:
Merge branch '4.4' into 5.0

* 4.4:
  Update user_checkers.rst

javiereguiluz committed:
Merge branch '5.0'

* 5.0:
  Update user_checkers.rst

javiereguiluz committed:
Merge branch '4.4' into 5.0

* 4.4:
  Update multiple_guard_authenticators.rst

javiereguiluz committed:
Merge branch '5.0'

* 5.0:
  Update multiple_guard_authenticators.rst

javiereguiluz committed:
minor #12865 Add documentation for new reverse() method (MrYamous)

This PR was merged into the master branch.

Discussion
----------

Add documentation for new reverse() method

PR for issue #12850

Adding documentation in master for new reverse() method

I was not sure whether it was necessary to create a new section or to add this method to an existing one, i'll edit according to your opinion if necessary.

Commits
-------

9a4d9cdba Add documentation for new reverse() method

javiereguiluz committed:
Minor tweaks

javiereguiluz committed:
Merge branch '4.4' into 5.0

* 4.4:
  Update custom_route_loader.rst | Fix wrong tag

javiereguiluz committed:
Merge branch '5.0'

* 5.0:
  Update custom_route_loader.rst | Fix wrong tag

javiereguiluz committed:
Merge branch '4.4' into 5.0

* 4.4:
  Fixed broken link

javiereguiluz committed:
Merge branch '5.0'

* 5.0:
  Fixed broken link

javiereguiluz committed:
Merge branch '4.4' into 5.0

* 4.4:
  Fixed problem normalizer example

javiereguiluz committed:
Merge branch '5.0'

* 5.0:
  Fixed problem normalizer example

javiereguiluz committed:
Merge branch '4.4' into 5.0

* 4.4:
  Update error_pages.rst

javiereguiluz committed:
Merge branch '5.0'

* 5.0:
  Update error_pages.rst

javiereguiluz committed:
Merge branch '4.4' into 5.0

* 4.4:
  Update error_handler.rst

javiereguiluz committed:
Merge branch '5.0'

* 5.0:
  Update error_handler.rst

javiereguiluz committed:
Merge branch '4.4' into 5.0

* 4.4:
  Update argument_value_resolver.rst

javiereguiluz committed:
Merge branch '5.0'

* 5.0:
  Update argument_value_resolver.rst

javiereguiluz committed:
minor #12844 Make custom routers compatible with the LoaderInterface (atailouloute)

This PR was merged into the 5.0 branch.

Discussion
----------

Make custom routers compatible with the LoaderInterface

Commits
-------

a49af0d72 Make custom routers compatible with the LoaderInterface

javiereguiluz committed:
Merge branch '5.0'

* 5.0:
  Make custom routers compatible with the LoaderInterface

javiereguiluz committed:
Merge branch '4.4' into 5.0

* 4.4:
  Update event_dispatcher.rst
  Update routing.rst

javiereguiluz committed:
Merge branch '5.0'

* 5.0:
  Update event_dispatcher.rst
  Update routing.rst

javiereguiluz committed:
Merge branch '4.4' into 5.0

* 4.4:
  Update doctrine.rst

javiereguiluz committed:
Merge branch '5.0'

* 5.0:
  Update doctrine.rst

javiereguiluz committed:
Merge branch '4.4' into 5.0

* 4.4:
  Add a note about voters with access_control

javiereguiluz committed:
Merge branch '5.0'

* 5.0:
  Add a note about voters with access_control

javiereguiluz committed:
Merge branch '4.4' into 5.0

* 4.4:
  Use HttpKernel Component

javiereguiluz committed:
Merge branch '5.0'

* 5.0:
  Use HttpKernel Component

javiereguiluz committed:
Merge branch '4.4' into 5.0

* 4.4:
  It's needed to change the returned string

javiereguiluz committed:
Merge branch '5.0'

* 5.0:
  It's needed to change the returned string

javiereguiluz committed:
Merge branch '4.4' into 5.0

* 4.4:
  Getting product using Symfony autowiring

javiereguiluz committed:
Merge branch '5.0'

* 5.0:
  Getting product using Symfony autowiring

javiereguiluz committed:
Merge branch '4.4' into 5.0

* 4.4:
  Update service id of the MailerInterface

javiereguiluz committed:
Merge branch '5.0'

* 5.0:
  Update service id of the MailerInterface

javiereguiluz committed:
Merge branch '4.4' into 5.0

* 4.4:
  Fix invalid yaml

javiereguiluz committed:
Merge branch '5.0'

* 5.0:
  Fix invalid yaml

wouterj committed:
Merge branch '4.4' into 5.0

* 4.4:
  [3.4] Fixed everything found by the spellcheck builder

wouterj committed:
Merge branch '5.0'

* 5.0:
  [3.4] Fixed everything found by the spellcheck builder

AnaniTH committed:
Fix a minor code mistake

fabpot committed:
Remove recommandation

javiereguiluz committed:
minor #12892 Remove recommendation (fabpot)

This PR was merged into the master branch.

Discussion
----------

Remove recommendation

We do not recommend to alias php to symfony anymore.

Commits
-------

10e6500c0 Remove recommandation

javiereguiluz committed:
Merge branch '4.4' into 5.0

* 4.4:
  Backported #12892 changes to 3.4
  Fix code example for definitionList.
  some spelling updates
  [DependencyInjection] Reworded the article about factories
  Fixed a wrong reference

javiereguiluz committed:
minor #12890 Fix a minor code mistake (AnaniTH)

This PR was merged into the 5.0 branch.

Discussion
----------

Fix a minor code mistake

Commits
-------

1e0ced5e0 Fix a minor code mistake

javiereguiluz committed:
Merge branch '5.0'

* 5.0:
  Backported #12892 changes to 3.4
  Fix a minor code mistake
  Fix code example for definitionList.
  some spelling updates
  [DependencyInjection] Reworded the article about factories
  Fixed a wrong reference

ogizanagi committed:
[Serializer] Remove note about CsvEncoder as_collection flag

This is the default since 5.0

lyrixx committed:
[Console] Updated section 'Building a single Command Application' with the new 'SingleCommandApplication'

JJarrie committed:
[Messenger] Redis example with Unix Socket

This is the documentation change for https://github.com/symfony/symfony/pull/35295

javiereguiluz committed:
[Notifier] Added a placeholder article for the component docs

Commit:
[FrameworkBundle] Template Controller should accept extra arguments fix #12929

javiereguiluz committed:
Merge branch '4.4' into 5.0

* 4.4:
  Fix dama/doctrine-test-bundle docs
  Update process.rst

javiereguiluz committed:
Merge branch '5.0'

* 5.0:
  Fix dama/doctrine-test-bundle docs
  Update process.rst

javiereguiluz committed:
Merge branch '4.4' into 5.0

* 4.4:
  Update extending.rst

javiereguiluz committed:
Merge branch '5.0'

* 5.0:
  Update extending.rst

javiereguiluz committed:
minor #12930 [FrameworkBundle] Template Controller should accept extra arguments (Benjamin RICHARD)

This PR was merged into the master branch.

Discussion
----------

[FrameworkBundle] Template Controller should accept extra arguments

fix #12929

Not sure of the format of the PR in this project, any help is welcome.

Commits
-------

bf737c386 [FrameworkBundle] Template Controller should accept extra arguments fix #12929

javiereguiluz committed:
Added the versionadded directive

javiereguiluz committed:
minor #12901 [Serializer] Remove note about CsvEncoder as_collection flag (ogizanagi)

This PR was merged into the 5.0 branch.

Discussion
----------

[Serializer] Remove note about CsvEncoder as_collection flag

This is the default since 5.0

https://github.com/symfony/symfony/blob/ad7d24109c5d9d9bce66ae57c509eb9baa3cc21b/src/Symfony/Component/Serializer/Encoder/CsvEncoder.php#L47
<!--

If your pull request fixes a BUG, use the oldest maintained branch that contains
the bug (see https://symfony.com/roadmap for the list of maintained branches).

If your pull request documents a NEW FEATURE, use the same Symfony branch where
the feature was introduced (and `master` for features of unreleased versions).

-->

Commits
-------

0be5b16cf [Serializer] Remove note about CsvEncoder as_collection flag

javiereguiluz committed:
Merge branch '5.0'

* 5.0:
  [Serializer] Remove note about CsvEncoder as_collection flag

javiereguiluz committed:
minor #12925 [Notifier] Added a placeholder article for the component docs (javiereguiluz)

This PR was squashed before being merged into the 5.0 branch (closes #12925).

Discussion
----------

[Notifier] Added a placeholder article for the component docs

Notifier doesn't have docs yet. We don't link to them from symfony.com ... but we link to them from the README of the code repo. So, let's add a placeholder article to avoid 404 errors.

Commits
-------

3095dff66 [Notifier] Added a placeholder article for the component docs

javiereguiluz committed:
Merge branch '5.0'

* 5.0:
  [Notifier] Added a placeholder article for the component docs

javiereguiluz committed:
Removed the versionadded directive

javiereguiluz committed:
Merge branch '4.4' into 5.0

* 4.4:
  remove getPublicDir() documentation

javiereguiluz committed:
Merge branch '5.0'

* 5.0:
  remove getPublicDir() documentation

javiereguiluz committed:
Merge branch '4.4' into 5.0

* 4.4:
  Update workflow-and-state-machine.rst

javiereguiluz committed:
Merge branch '5.0'

* 5.0:
  Update workflow-and-state-machine.rst

javiereguiluz committed:
Merge branch '4.4' into 5.0

* 4.4:
  Hint for generating container preload file

javiereguiluz committed:
Merge branch '5.0'

* 5.0:
  Hint for generating container preload file

javiereguiluz committed:
Merge branch '4.4' into 5.0

* 4.4:
  Fixes unclear sentence in finder.rst
  Fix lock.invoice.retry_till_save.store decoration

javiereguiluz committed:
Merge branch '5.0'

* 5.0:
  Fixes unclear sentence in finder.rst
  Fix lock.invoice.retry_till_save.store decoration

lyrixx committed:
[workflow] Added a way to not fire the announce event

javiereguiluz committed:
Merge branch '4.4' into 5.0

* 4.4:
  adding the * to .xml in the Conf Env

javiereguiluz committed:
Merge branch '5.0'

* 5.0:
  adding the * to .xml in the Conf Env

javiereguiluz committed:
minor #12942 [workflow] Added a way to not fire the announce event (lyrixx)

This PR was squashed before being merged into the master branch (closes #12942).

Discussion
----------

[workflow] Added a way to not fire the announce event

fixes #12941

Commits
-------

57ac25dfa [workflow] Added a way to not fire the announce event

javiereguiluz committed:
Added the versionadded directive

OskarStark committed:
feature #12917 [Console] Updated section 'Building a single Command Application' with the new 'SingleCommandApplication' (lyrixx)

This PR was merged into the master branch.

Discussion
----------

[Console] Updated section 'Building a single Command Application' with the new 'SingleCommandApplication'

fixes #12903

Commits
-------

70d523c7c [Console] Updated section 'Building a single Command Application' with the new 'SingleCommandApplication'

OskarStark committed:
minor. refs #12917

OskarStark committed:
Merge branch '4.4' into 5.0

* 4.4:
  Update extending.rst
  Update example to use tagged_locator helper function

OskarStark committed:
Merge branch '5.0'

* 5.0:
  Update extending.rst
  Update example to use tagged_locator helper function

OskarStark committed:
Merge branch '4.4' into 5.0

* 4.4:
  minor
  Move method and class
  Use the get method instead of using the array

OskarStark committed:
Merge branch '5.0'

* 5.0:
  minor
  Move method and class
  Use the get method instead of using the array

javiereguiluz committed:
[FrameworkBundle] Mention that setting router.utf8 is now mandatory

javiereguiluz committed:
[Yaml] Documented the yaml-lint binary

OskarStark committed:
Merge branch '4.4' into 5.0

* 4.4:
  Fix Typo

OskarStark committed:
Merge branch '5.0'

* 5.0:
  Fix Typo

javiereguiluz committed:
minor #12948 [FrameworkBundle] Mention that setting router.utf8 is now mandatory (javiereguiluz)

This PR was squashed before being merged into the master branch (closes #12948).

Discussion
----------

[FrameworkBundle] Mention that setting router.utf8 is now mandatory

This fixes #12936.

Commits
-------

51cf7f069 [FrameworkBundle] Mention that setting router.utf8 is now mandatory

javiereguiluz committed:
Merge branch '4.4' into 5.0

* 4.4:
  Fixed an RST syntax issue

javiereguiluz committed:
Merge branch '5.0'

* 5.0:
  Fixed an RST syntax issue

javiereguiluz committed:
minor #12952 [Yaml] Documented the yaml-lint binary (javiereguiluz)

This PR was squashed before being merged into the master branch (closes #12952).

Discussion
----------

[Yaml] Documented the yaml-lint binary

Fixes #12935.

Commits
-------

b6dcce00f [Yaml] Documented the yaml-lint binary

javiereguiluz committed:
Merge branch '4.4' into 5.0

* 4.4:
  Fix invalid variable name

javiereguiluz committed:
Merge branch '5.0'

* 5.0:
  Fix invalid variable name

javiereguiluz committed:
Merge branch '4.4' into 5.0

* 4.4:
  tweaking the password upgrading functionality

javiereguiluz committed:
Merge branch '5.0'

* 5.0:
  tweaking the password upgrading functionality

javiereguiluz committed:
Merge branch '4.4' into 5.0

* 4.4:
  Remove usages of transChoice() method for Symfony 5.0

javiereguiluz committed:
Merge branch '5.0'

* 5.0:
  Remove usages of transChoice() method for Symfony 5.0

l-vo committed:
Update service_container.rst

Controller base class doesn't exist anymore in SF 5.0

javiereguiluz committed:
Merge branch '4.4' into 5.0

* 4.4:
  Fixed typo

javiereguiluz committed:
Merge branch '5.0'

* 5.0:
  Fixed typo

javiereguiluz committed:
minor #12977 Update service_container.rst (l-vo)

This PR was merged into the 5.0 branch.

Discussion
----------

Update service_container.rst

Controller base class doesn't exist anymore in SF 5.0

Commits
-------

3d75d9a7d Update service_container.rst

javiereguiluz committed:
Merge branch '5.0'

* 5.0:
  Update service_container.rst

pierredup committed:
Fix typo in notifier component

javiereguiluz committed:
Merge branch '4.4' into 5.0

* 4.4:
  Updated the intro of the Twig extensions article

javiereguiluz committed:
minor #12987 Fix typo in notifier component (pierredup)

This PR was merged into the 5.0 branch.

Discussion
----------

Fix typo in notifier component

<!--

If your pull request fixes a BUG, use the oldest maintained branch that contains
the bug (see https://symfony.com/roadmap for the list of maintained branches).

If your pull request documents a NEW FEATURE, use the same Symfony branch where
the feature was introduced (and `master` for features of unreleased versions).

-->

Commits
-------

905caae7d Fix typo in notifier component

javiereguiluz committed:
Merge branch '5.0'

* 5.0:
  Fix typo in notifier component
  Updated the intro of the Twig extensions article

javiereguiluz committed:
Merge branch '4.4' into 5.0

* 4.4:
  Adding details about recipe upgrade system and upgrade improvements
  Updated link

javiereguiluz committed:
Merge branch '5.0'

* 5.0:
  Adding details about recipe upgrade system and upgrade improvements
  Updated link

javiereguiluz committed:
minor #12924 [Messenger] Redis example with Unix Socket (JJarrie)

This PR was merged into the master branch.

Discussion
----------

[Messenger] Redis example with Unix Socket

This is the documentation change for https://github.com/symfony/symfony/pull/35295 .

Commits
-------

13af19487 [Messenger] Redis example with Unix Socket

javiereguiluz committed:
Added the versionadded directive

OskarStark committed:
Merge branch '4.4' into 5.0

* 4.4:
  Fix build. Refs #12959

OskarStark committed:
Merge branch '5.0'

* 5.0:
  Fix build. Refs #12959

weaverryan committed:
Merge branch '4.4' into 5.0

* 4.4:
  tweaks thanks to review
  finishing the secrets documentation
  Add the secret documentation

weaverryan committed:
Merge branch '5.0'

* 5.0:
  tweaks thanks to review
  finishing the secrets documentation
  Add the secret documentation

javiereguiluz committed:
Merge branch '4.4' into 5.0

* 4.4:
  Update authorization.rst

javiereguiluz committed:
Merge branch '5.0'

* 5.0:
  Update authorization.rst

javiereguiluz committed:
Merge branch '4.4' into 5.0

* 4.4:
  [Security] Fixed the name of a method

javiereguiluz committed:
Merge branch '5.0'

* 5.0:
  [Security] Fixed the name of a method

javiereguiluz committed:
Merge branch '4.4' into 5.0

* 4.4:
  Updated imports in TaskController when using TaskType

javiereguiluz committed:
Merge branch '5.0'

* 5.0:
  Updated imports in TaskController when using TaskType

Livda committed:
[Mailer] add ability to disable the TLS peer verification via DSN

javiereguiluz committed:
Merge branch '4.4' into 5.0

* 4.4:
  composer issue for prefer-lowest has been fixed

javiereguiluz committed:
Merge branch '5.0'

* 5.0:
  composer issue for prefer-lowest has been fixed

Nyholm committed:
[Messenger] Document Redis TLS support

Nyholm committed:
Remove old versionadded

OskarStark committed:
minor #13007 Remove old versionadded (Nyholm)

This PR was submitted for the master branch but it was merged into the 5.0 branch instead (closes #13007).

Discussion
----------

Remove old versionadded

Build failed in #13006 because of these.

They are unrelated and I assume they should be removed.

Commits
-------

ff2b2fc63 Remove old versionadded

OskarStark committed:
Merge branch '5.0'

* 5.0:
  Remove old versionadded

OskarStark committed:
feature #13005 [Messenger] Document Redis TLS support (Nyholm)

This PR was merged into the master branch.

Discussion
----------

[Messenger] Document Redis TLS support

This will fix #13003

Commits
-------

37ce66cad [Messenger] Document Redis TLS support

javiereguiluz committed:
minor #12997 [Mailer] add ability to disable the TLS peer verification via DSN (Aurélien Fontaine)

This PR was squashed before being merged into the master branch (closes #12997).

Discussion
----------

[Mailer] add ability to disable the TLS peer verification via DSN

Documentation for the PR https://github.com/symfony/symfony/pull/35262

Commits
-------

28a391a10 [Mailer] add ability to disable the TLS peer verification via DSN

javiereguiluz committed:
Reword

OskarStark committed:
Merge branch '4.4' into 5.0

* 4.4:
  Workflow: align Example for property access with implementation of Me…

OskarStark committed:
Merge branch '5.0'

* 5.0:
  Workflow: align Example for property access with implementation of Me…

OskarStark committed:
Merge branch '4.4' into 5.0

* 4.4:
  Minor
  Minor. refs #12949
  [Translation] Added tip about the incompatibility of % character

OskarStark committed:
Merge branch '5.0'

* 5.0:
  Minor
  Minor. refs #12949
  [Translation] Added tip about the incompatibility of % character

OskarStark committed:
Merge branch '4.4' into 5.0

* 4.4:
  fix XML config example

OskarStark committed:
Merge branch '5.0'

* 5.0:
  fix XML config example

wouterj committed:
Merge branch '4.4' into 5.0

* 4.4:
  Promote usage of a single translation domain
  Move sidebar into a real section
  Removed Translation component documentation

wouterj committed:
Merge branch '5.0'

* 5.0:
  Promote usage of a single translation domain
  Move sidebar into a real section
  Removed Translation component documentation

wouterj committed:
Merge branch '4.4' into 5.0

* 4.4:
  [#12861] Tweaked translation docs for 4.4 component changes
  [#12861] Tweak translation articles for 4.3 component changes

wouterj committed:
Merge branch '5.0'

* 5.0:
  [#12861] Tweaked translation docs for 4.4 component changes
  [#12861] Tweak translation articles for 4.3 component changes

wouterj committed:
Merge branch '4.4' into 5.0

* 4.4:
  Removed leftover 3.4 versionadded directive

wouterj committed:
Merge branch '5.0'

* 5.0:
  Removed leftover 3.4 versionadded directive

wouterj committed:
Merge branch '4.4' into 5.0

* 4.4:
  Added the other two removed component articles to the redirection map

wouterj committed:
Merge branch '5.0'

* 5.0:
  Added the other two removed component articles to the redirection map

wouterj committed:
Merge branch '4.4' into 5.0

* 4.4:
  Fixed missing reference error

wouterj committed:
Merge branch '5.0'

* 5.0:
  Fixed missing reference error

wouterj committed:
Merge branch '4.4' into 5.0

* 4.4:
  Fixed deadlink to Translation components docs

wouterj committed:
Merge branch '5.0'

* 5.0:
  Fixed deadlink to Translation components docs

OskarStark commented:
Thanks for your work on this new feature!
